featuretools/primitives/standard/binary_transform.py

```diff
@@ -230,7 +230,9 @@ class SubtractNumeric(TransformPrimitive):
     name = "subtract_numeric"
```

Comment:
can we add a docstring to this class? Something like this

```
"""Subtracts one feature from another and returns the difference
    Args:
        commutative (bool): determines if Deep Feature Synthesis should generate
            both a - b and b - a, or just one.  If commutative is False, there is no
            guarantee which of the two will be generated. Defaults to True.
"""
```

Reply:
it looks strange to add `docstring` to `SubtractNumeric` and `DivideNumeric` classes only. All other primitives don't have docstrings. What do you think?

Reply:
it's only in binary_transform.py that don't have docstrings. the other primitive files do. 

we didn't put them here since most people aren't importing and using the binary primitives directly. however, now that you added a parameter, someone will need to know how to use it. 

if you have time, you're more than welcome to add docstrings to the other methods in binary_transform.py!